help/destinations/catalog/advertising/amazon-ads.md

```diff
@@ -0,0 +1,133 @@
+
+
+
+* **[!UICONTROL Bearer token]**: Fill in the bearer token to authenticate to the destination.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Amazon Ads Advertiser Id]**: Select the ID for the target Amazon Ads account used for the destination.
+
+Note: once selecting this Amazon Ads Advertiser Id, you will need to create a new destination to change this. If you re-authenticate the OAuth credentials and select a new Advertiser Id, your changes will not apply.
+
+![Configure new destination](../../assets/catalog/advertising/amazon_ads_image_1.png)
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
+
+The Amazon Ads connection supports hashed email address and hashed phone numbers for identity matching purposes.  The screenshot below provides an example matching that is compatible with the Amazon Ads connection:
+
+![Adobe to Amazon Ads mapping](../../assets/catalog/advertising/amazon_ads_image_2.png)
+
+* When mapping email, select the Lower case, SHA256 source field (Email_LC_SHA256)
```

Comment:
To map hashed email addresses, select the `Email_LC_SHA256` identity namespace as a source field.